--- a/src/wannier/m_rdinput.f90
+++ b/src/wannier/m_rdinput.f90
@@ -111,13 +111,13 @@
 endif 
 !--
 !default set_inner_window==F
-if(set_inner_window==.true..and.E_UPPER_inner==0.0d0.and.E_LOWER_inner==0.0d0)then 
+if(set_inner_window.eqv..true..and.E_UPPER_inner==0.0d0.and.E_LOWER_inner==0.0d0)then 
  write(6,'(a)')'wrong input: inner energy windows; stop'
  write(6,'(a)')'Lower_inner_window = Upper_inner_window = 0.0d0: WRONG'
  write(6,'(a,2f20.9)')'Upper_inner_window,Lower_inner_window',Upper_inner_window,Lower_inner_window
  stop
 endif 
-if(set_inner_window==.true.)then 
+if(set_inner_window.eqv..true.)then 
  if(E_LOWER_inner>=E_UPPER_inner)then 
   write(6,'(a)')'wrong input: inner energy windows; stop'
   write(6,'(a)')'Lower_inner_window >= Upper_inner_window: WRONG'
