--- a/src/chiqw/chiqw.F
+++ b/src/chiqw/chiqw.F
@@ -525,7 +525,7 @@
 !20170418 
 !mkdir dir-eps/qxx
       if(myrank_glb.eq.0)then!master 
-       inquire(directory='./dir-eps',exist=dir_e) 
+       inquire(file="./dir-eps",exist=dir_e) 
        if(dir_e)then
         write(6,*) 'dir exist'
        else
@@ -536,7 +536,7 @@
        call system('pwd') 
        do ik=1,Nk_irr 
         write(dirname,"('q',i3.3)")ik 
-        inquire(directory=dirname,exist=dir_e) 
+        inquire(file=dirname,exist=dir_e) 
         if(dir_e)then
          write(6,*) 'dir exist'
         else
